File: main/src/main/java/net/fwitz/math/main/binary/split/functions/examples/simple/SplitInverseZ.java
package net.fwitz.math.main.binary.split.functions.examples.simple;

import net.fwitz.math.binary.split.SplitComplex;
import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot;

public class SplitInverseZ {
    public static void main(String[] args) {
        new SplitComplexFunctionPlot("1/z")
                .computeFn(SplitComplex::inverse)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/split/functions/examples/simple/SplitInverseZ.kt
package net.fwitz.math.main.binary.split.functions.examples.simple

import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot

object SplitInverseZ {
    @JvmStatic
    fun main(args: Array<String>) = SplitComplexFunctionPlot("1/z")
        .computeFn { z -> z.inverse }
        .render()
}